tasks/main.yml

```diff
@@ -4,20 +4,20 @@
     name: sudo
     update_cache: yes
     state: present
-  when: "{{ ansible_os_family == 'RedHat' }}"
+  when: ansible_os_family == 'RedHat'
```

Comment:
In order to make this following best practices can you wrap it with template tags?